src/main/resources/hudson/plugins/promoted_builds/PromotionProcess/process-config.jelly

```diff
@@ -41,6 +41,11 @@
         </select>
       </f:entry>
 
+      <f:entry title="Any_Met">
```

Comment:
Is this what shows up in the UI? Maybe something like `Promote if ANY of the above conditions are met` would be more clear to users, but I'm not sure what would be best.

Reply:
The GUI looks like this:
Due to the alignment, I can't have long string at the title.
![image](https://user-images.githubusercontent.com/1186506/44287637-614faf00-a222-11e8-930a-38d6d6f45755.png)

Reply:
Thanks for the screenshot! Maybe use `Any Met` with no underscore?